pkgs/build-support/build-fhs-userenv/env.nix

```diff
@@ -52,8 +52,8 @@ let
   etcProfile = writeText "profile" ''
     export PS1='${name}-chrootenv:\u@\h:\w\$ '
     export LOCALE_ARCHIVE='/usr/lib/locale/locale-archive'
-    export LD_LIBRARY_PATH='/run/opengl-driver/lib:/run/opengl-driver-32/lib:/usr/lib:/usr/lib32'
-    export PATH='/run/wrappers/bin:/usr/bin:/usr/sbin'
+    export LD_LIBRARY_PATH="/run/opengl-driver/lib:/run/opengl-driver-32/lib:/usr/lib:/usr/lib32:$LD_LIBRARY_PATH"
```

Comment:
Lose the `"`s.

Reply:
Care to elaborate? I would keep them as they don't cause any harm (and I personally prefer quoting everything by default). While it shouldn't make a difference for Bash, there are or at least where shell where it does make a difference (IIRC).

Reply:
Section 2.3 Token Recognition from http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_03 specifies this.

Reply:
Doesn't really matter much, I'll merge